dbms/src/main/java/org/polypheny/db/routing/strategies/CreatePlacementStrategy.java

```diff
@@ -18,16 +18,16 @@
 
 import java.util.List;
 import org.polypheny.db.adapter.DataStore;
-import org.polypheny.db.catalog.entity.CatalogColumn;
+import org.polypheny.db.catalog.entity.logical.LogicalColumn;
 
 
 /**
- * Interface to define, on which store(s) tables and columns are created.
+ * Interface to define, on which storeId(s) tables and columns are created.
```

Comment:
Looks like a find and replace error. Comments and method / variable names in this class are relational

Reply:
Fixed, and renamed wherever sensible.